marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/provenance/migrateProvenance/migrateProvenanceTest.sjs

```diff
@@ -0,0 +1,31 @@
+const config = require("/com.marklogic.hub/config.sjs");
+const hubTest = require("/test/data-hub-test-helper.sjs");
+const hubTestXqy = require("/test/data-hub-test-helper.xqy");
+const test = require("/test/test-helper.xqy");
+const provenanceService = require("../../lib/provenanceService.sjs");
+
+const assertions = [];
+
+function migrateProvAsOperator() {
+  hubTest.runWithRolesAndPrivileges(['data-hub-operator', 'ps-user'], [], function() {
+    provenanceService.migrateProvenance({}, {});
+  });
+}
+
+assertions.push(test.assertEqual(0, hubTest.getProvenanceCount(config.STAGINGDATABASE), "There should be 0 provenance documents in the Staging database"));
+assertions.push(test.assertEqual(0, hubTest.getProvenanceCount(config.FINALDATABASE), "There should be 0 provenance documents in the Staging database"));
+assertions.push(test.assertEqual(5, hubTest.getProvenanceCount(), "There should be 5 provenance documents in the Jobs database"));
+
+migrateProvAsOperator();
+
+assertions.push(test.assertEqual(2, hubTest.getProvenanceCount(config.STAGINGDATABASE), "There should now be 2 provenance documents in the Staging database"));
+assertions.push(test.assertEqual(3, hubTest.getProvenanceCount(config.FINALDATABASE), "There should now be 2 provenance documents in the Staging database"));
```

Comment:
Small typo, should be 3 instead of 2 in the message and Staging -> Final

Reply:
fixed